learn_script.py

```diff
@@ -45,8 +61,7 @@ def query_user(content):
         user_input = fancy_input()
     while not re.match(content, user_input):
         ansiprint(
-            "<fg red>Certeza que deseja continuar? A entrada parece ser diferente do indicado.</fg red> <bold>[s/N]</bold> ",
-            end="",
+            "<fg red>Certeza que deseja continuar? A entrada parece ser diferente do indicado.</fg red> <bold>[s/N]</bold> "
```

Comment:
keep end of line empty here. we want to simulate most default query behavior (asking the question and waiting for an answer on the same line).

```suggestion
            "<fg red>Certeza que deseja continuar? A entrada parece ser diferente do indicado.</fg red> <bold>[s/N]</bold> ",
            end="",
```